scripts/drone/steps/lib.star

```diff
@@ -510,18 +510,30 @@ def test_backend_step(edition):
         ],
     }
 
-
 def test_backend_integration_step(edition):
-    return {
-        'name': 'test-backend-integration' + enterprise2_suffix(edition),
-        'image': build_image,
-        'depends_on': [
-            'wire-install',
-        ],
-        'commands': [
-            './bin/grabpl integration-tests --edition {}'.format(edition),
-        ],
-    }
+    if edition == 'oss':
+        return {
+            'name': 'test-backend-integration',
+            'image': build_image,
+            'depends_on': [
+                'wire-install',
+            ],
+            'commands': [
+                'go test -run Integration -covermode=atomic -timeout=30m ./pkg/...',
+            ],
+        }
+    else:
+        return {
+            'name': 'test-backend-integration' + enterprise2_suffix(edition),
+            'image': build_image,
+            'depends_on': [
+                'wire-install',
+            ],
+            'commands': [
+                './bin/grabpl integration-tests --edition {}'.format(edition),
```

Comment:
This won't run the integration tests for OSS when on enterprise if the tags aren't there.

Should we just make an equivalent PR in enterprise to remove the tags instead of running grabpl in enterprise and go test in OSS?